src/pretix/base/services/cart.py

```diff
@@ -590,6 +612,7 @@ def extend_expired_positions(self):
                 self._voucher_use_diff[cp.voucher] += 2
 
             self._operations.append(op)
+            self.num_extended_positions += 1
```

Comment:
Would it be more "correct" to increase this only after *execution* of the ExtendOperation? Otherwise the `num_extended_positions` of a CartManager with a failed commit has a weird semantic.